src/shared/dropdown-list/dropdown-list.styles.tsx

```diff
@@ -246,10 +272,20 @@ export const ResultStateContainer = styled.div`
 `;
 
 export const ResultStateText = styled(Text.Body)``;
+export const ResultStateTextSmall = styled(Text.BodySmall)``;
 
-export const LabelIcon = styled(ExclamationCircleFillIcon)`
+export const LabelIcon = styled(ExclamationCircleFillIcon)<IconProps>`
+    ${(props) => {
+        let _size = 1.5;
```

Comment:
no need to prefix variables

```suggestion
        let size = 1.5;
```